### ISSUE TITLE
Add possibility to choose a provider as default

### ISSUE THREAD

**Issue by oanalavinia** [xwikisas/identity-oauth#19]
Prerequisites: Have your own provider (or multiple) configured as explained on https://github.com/xwikisas/identity-oauth/blob/master/README.md

Right now, when you want to login using a provider you have to first go to the login page and then chose the provider. 
Most of the time, only a specific provider is used and displaying the login page with options can be a redundant step, considering also that after #18 is implemented, it might display a single option:
![image](https://user-images.githubusercontent.com/22794181/200581090-3f2e24e2-f218-4e66-ba01-c13544791fb4.png)
For this, it would be nice to have the possibility to configure a specific provider to be used by default. This way, the provider's login page will be displayed directly.

**Comment by oanalavinia** [comment]
A new configuration was added as part of `Administer Wiki -> Other -> Identity OAuth` . You can now select the hint of the provider that should be forced on the login page
![image](https://github.com/xwikisas/identity-oauth/assets/22794181/f3616f44-8440-4554-bfc2-5295f7a55209)
